stankove_napady.md

```diff
@@ -0,0 +1,105 @@
+Zkusim popsat svuj navrh a zkusme to spolecnymi pripominkami upravit do lepsi podoby. Je to moje stankova premiera.
+<br>
+</br>
+## Co chceme prezentovat?
+
+**python.cz** 
+
+> aby je navstevovalo vice pythonistu i zacatecniku zajemcu o python
+
+> aby vedeli jake informace tam najdou
+
+**pyvo.cz**
```

Comment:
Navrhuji tam někde mít vytištěnou mapku z úvodní stránky Pyvo.cz, vypadá to hezky.